keras/ops/numpy.py

```diff
@@ -153,6 +153,9 @@
 from keras.ops.operation import Operation
 from keras.ops.operation_utils import reduce_shape
 
+# save reference to built in max operation before redefinition
+_max = max  # pylint:disable=used-before-assignment
```

Comment:
Hi @jackd 
I believe you can use `import builtins` to avoid redefinition

For example:
https://github.com/keras-team/keras/blob/b6df549577a67c98ab1ce4df5c74681fcc98efa1/keras/backend/torch/numpy.py#L1322

Reply:
Much cleaner, thanks!